cmd/root.go

```diff
@@ -61,18 +61,21 @@ func New() *cobra.Command {
 	cmd.AddCommand(stop.New(&o.Cloud))
 
 	cmd.PersistentFlags().StringVar(&o.Provider, "cloud", "", "cloud provider")
-	cmd.PersistentFlags().StringVar(&o.Log, "log", "info", "log level")
+	cmd.PersistentFlags().CountVar(&o.Verbose, "verbose", "verbose output")
```

Comment:
Why `CountVar` and not just `BoolVar`?

Reply:
Just in case we introduced additional debug levels and a `-v` shorthand, so we can act differently upon `-v`, `-vv`, `-vvv`, ...

Reply:
We can use a `BoolVar` until that functionality is really needed, though. It was probably a premature design choice.

Reply:
I think we can should make it a `BoolVar` for now.

Reply:
Resolved with  304c496